issue found on: mock.go:76:2: importShadow

an issue has been detected in source code: ./mock.go:76:2: importShadow: shadow of imported from 'github.com/RedHatInsights/insights-results-aggregator-mock/content' package 'content'

--- a/mock.go
+++ b/mock.go
@@ -73,7 +73,7 @@ func startService(config conf.ConfigStruct) int {
 		return ExitStatusServerError
 	}
 
-	content, err := content.ParseContent(contentCfg.Path)
+	ruleContent, err := content.ParseContent(contentCfg.Path)
 	if err != nil {
 		log.Error().Err(err).Msg("Content init error")
 		return ExitStatusServerError
@@ -86,7 +86,7 @@ func startService(config conf.ConfigStruct) int {
 		return ExitStatusServerError
 	}
 
-	serverInstance = server.New(serverCfg, storageInstance, groups, content)
+	serverInstance = server.New(serverCfg, storageInstance, groups, ruleContent)
 
 	err = serverInstance.Start()
 	if err != nil {

--- a/mock.go
+++ b/mock.go
@@ -78,7 +78,7 @@ func startService(config conf.ConfigStruct) int {
 		log.Error().Err(err).Msg("Content init error")
 		return ExitStatusServerError
 	}
-	log.Info().Int("count", len(content)).Msg("Content read")
+	log.Info().Int("count", len(ruleContent)).Msg("Content read")
 
 	storageInstance, err := storage.New(config.Paths.MockDataPath)
 	if err != nil {

